pkg/config/testdata/TestLoadedConfigValidationErrors/app-token-only.yaml

```diff
@@ -12,3 +12,7 @@ communications: # req 1 elm.
               - k8s-events
 
       appToken: 'xapp-SLACK_API_TOKEN'
+executors:
+  kubectl-read-only:
```

Comment:
IMO this would look nicer 😄 

```
kubectl-read-only: {}
```

If you agree, please change it in all places. Thanks!